src/connector/src/parser/avro/parser.rs

```diff
@@ -261,290 +251,4 @@ mod test {
-        null_record.put("score", Value::Union(1, Box::new(Value::Null)));
-        null_record.put("avg_score", Value::Union(0, Box::new(Value::Null)));
-        null_record.put("is_lasted", Value::Union(0, Box::new(Value::Null)));
-        null_record.put("entrance_date", Value::Union(0, Box::new(Value::Null)));
-        null_record.put("birthday", Value::Union(0, Box::new(Value::Null)));
-        null_record.put("anniversary", Value::Union(0, Box::new(Value::Null)));
-
-        let mut writer = Writer::new(schema, Vec::new());
-        writer.append(null_record).unwrap();
-        writer.flush().unwrap();
-
-        let record = build_avro_data(schema);
-        writer.append(record).unwrap();
-        writer.flush().unwrap();
-
-        let records = writer.into_inner().unwrap();
-
-        let reader: Vec<_> = Reader::with_schema(schema, &records[..]).unwrap().collect();
-        assert_eq!(2, reader.len());
-        let null_record_expected: Vec<(String, Value)> = vec![
-            ("id".to_string(), Value::Int(5)),
-            ("age".to_string(), Value::Union(0, Box::new(Value::Null))),
-            (
-                "sequence_id".to_string(),
-                Value::Union(0, Box::new(Value::Null)),
-            ),
-            ("name".to_string(), Value::Union(0, Box::new(Value::Null))),
-            ("score".to_string(), Value::Union(1, Box::new(Value::Null))),
-            (
-                "avg_score".to_string(),
-                Value::Union(0, Box::new(Value::Null)),
-            ),
-            (
-                "is_lasted".to_string(),
-                Value::Union(0, Box::new(Value::Null)),
-            ),
-            (
-                "entrance_date".to_string(),
-                Value::Union(0, Box::new(Value::Null)),
-            ),
-            (
-                "birthday".to_string(),
-                Value::Union(0, Box::new(Value::Null)),
-            ),
-            (
-                "anniversary".to_string(),
-                Value::Union(0, Box::new(Value::Null)),
-            ),
-        ];
-        let null_record_value = reader.first().unwrap().as_ref().unwrap();
-        match null_record_value {
-            Value::Record(values) => {
-                assert_eq!(values, &null_record_expected)
-            }
-            _ => unreachable!(),
-        }
-    }
-
-    // run this script when updating `simple-schema.avsc`, the script will generate new value in
-    // `avro_simple_schema_bin.1`
-    #[ignore]
-    #[tokio::test]
-    async fn update_avro_payload() {
-        let conf = new_avro_conf_from_local("simple-schema.avsc")
-            .await
-            .unwrap();
-        let mut writer = Writer::new(conf.schema.original_schema.as_ref(), Vec::new());
-        let record = build_avro_data(conf.schema.original_schema.as_ref());
-        writer.append(record).unwrap();
-        let encoded = writer.into_inner().unwrap();
-        println!("path = {:?}", e2e_file_path("avro_simple_schema_bin.1"));
```

Comment:
There're still some usage of this data file to be cleaned.
<img width="341" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/37948597/73f7dc21-8988-4141-b57b-a35a38b2431b">